deploy/cdk/lib/user-content/index.ts

```diff
@@ -85,6 +85,11 @@ export class UserContentStack extends cdk.Stack {
     // API Gateway
     const api = new apigateway.RestApi(this, 'userContentApi', {
       restApiName: 'Marble User Content Service',
+      defaultCorsPreflightOptions: {
+        allowOrigins: ['*'],
```

Comment:
Are you sure you don't want to limit this to a domain?

Reply:
I've moved this into a variable so it can be changed through context as suggested. Currently still defaults to '*'.